src/components/InfiniteScroll/InfiniteScroll.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { Component } from 'react';
```

Comment:
filename is wrong, just rename it as `InfiniteScroll/index.jsx`

Reply:
ok